test/DynamoCoreWpfTests/SearchSideEffects.cs

```diff
@@ -22,25 +22,24 @@ public void WhenStartingDynamoInputAndOutputNodesAreNolongerMissingFromSearch()
             Assert.AreEqual(1, ViewModel.CurrentSpaceViewModel.InCanvasSearchViewModel.FilteredResults.Count(x => x.Model.Name == "Output"));
         }
 
-
         /// <summary>
         /// This test will validate that the nodes "Input", "Output", "And", "Or", "Not", "+", "-"  appear in the InCanvasSearch results
         /// </summary>
         [Test]
-        [Category("UnitTests")]
```

Comment:
@RobertGlobant20 [Category("UnitTests")] should also be present.

Reply:
I searched for Failure category in Dynamo solution and in all the places is only Failure, do you have an example of a test with both categories?
e.g.
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/faadab84-5dcc-4d92-975c-2059f527cfe0)

Reply:
You can just add another category attribute like this: https://github.com/DynamoDS/Dynamo/blob/master/test/DynamoCoreTests/UnitsOfMeasureTests.cs#L1047

Reply:
@reddyashish ok got your point.
Due that this PR is a cherry-pick, could you approve and merge it? (If I do changes over this branch won't be cherry-picking the commit).
I will do the change in the original branch and then create another cherry-pick PR.

Reply:
This also works